src/bindings/python/docs/test_examples.md

```diff
@@ -3,18 +3,28 @@
 #### Building and environment
 Instructions can be found in ["Building the OpenVINO™ Python API"](./build.md).
 
-### Running OpenVINO™ Python API tests
+Install the specific requirements file for testing:
+```
+python -m pip install -r openvino/src/bindings/python/requirements_test.txt
+```
+
+### Run OpenVINO™ Python API tests
 *For simplicity, all of these commands require to navigate to the main Python API folder first:*
 ```shell
 cd .../openvino/src/bindings/python/
 ```
 
-To run **all tests**:
+To run OpenVINO Python API 2.0 tests:
 ```shell
 pytest tests/
 ```
 
-Test framework *pytest* allows to filter tests with `-k` flag.
+To run OpenVINO Python API 1.0 tests, use this command:
+```
+pytest tests_compatibility/
+```
+
+The *pytest* test framework enables you to filter tests with the `-k` flag.
```

Comment:
should we add how to run tests on GPU or other accelerator device instead of  default CPU?

Reply:
thanks for reminding me, added